oresat_dxwifi/services/camera.py

```diff
@@ -0,0 +1,95 @@
+"""Oresat Live Camera Service"""
+
+from olaf import Service, logger
+from enum import IntEnum
+from ..camera.camera import CameraInterface
+
+
+class State(IntEnum):
+    OFF = 0
+    BOOT = 1
+    STANDBY = 2
+    FILMING = 3
+    ERROR = 0xFF
+
+
+STATE_TRANSISTIONS = {
+    State.OFF: [State.BOOT],
+    State.BOOT: [State.STANDBY],
+    State.STANDBY: [State.FILMING],
+    State.FILMING: [State.STANDBY, State.ERROR],
+    State.ERROR: [],
+}
+
+
+class CameraService(Service):
+    def __init__(self):
+        super().__init__()
+        self.state = State.BOOT
+
+        self.spv = 3  # Should evenly divide into duration
```

Comment:
A lot of these parameters are redundant -- we're setting members of `CameraService` but never using them, except to instantiate the `self.camera` member.

We could just pass these directly as arguments to the `CameraInterface` constructor.

Reply:
A lot of this stuff still needs cleanup because it tried to maintain the integrity of oresat-live-software/camera cmd functionality.